AEPAssurance/Source/AssuranceSessionOrchestrator.swift

```diff
@@ -99,12 +120,17 @@ class AssuranceSessionOrchestrator: AssurancePresentationDelegate {
     func canProcessSDKEvents() -> Bool {
         return session != nil || outboundEventBuffer != nil
     }
+    
+    // MARK: - AssurancePresentationDelegate
+    func initializePinScreenFlow() {
```

Comment:
I'd like to see delegate method implementations moved to their own extension (can be in the same file) -- makes the code much easier to read.